test/box-luatest/gh_9923_invalid_iproto_index_name_mapping_on_the_server_side_test.lua

```diff
@@ -0,0 +1,87 @@
+local net = require('net.box')
+local server = require('luatest.server')
+local t = require('luatest')
+
+local g = t.group()
+
+g.before_all(function()
+    g.server = server:new({alias = 'master'})
+    g.server:start()
+    g.server:exec(function()
+        box.schema.create_space('test', {
+            format = {
+                {name = "id", type = "unsigned"},
+                {name = "name", type = "string"},
+            },
```

Comment:
Nit: do you actually rely on the space format in the test? I think you can safely drop this as redundant.